frontend/src/pages/edgeMVP/EdgeModelRegistry.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { Bullseye } from '@patternfly/react-core';
+import ApplicationsPage from '~/pages/ApplicationsPage';
+import { useWatchComponents } from '~/utilities/useWatchComponents';
+import EmptyModelRegistry from './EmptyModelRegistry';
+
+type EdgeModelRegistryInnerProps = {
+  loaded: boolean;
+};
+
+const EdgeModelRegistryInner: React.FC<EdgeModelRegistryInnerProps> = React.memo(({ loaded }) => (
+  <ApplicationsPage title="Model registry" description={''} loaded={loaded} empty={false}>
+    <Bullseye>
+      <EmptyModelRegistry />
+    </Bullseye>
+  </ApplicationsPage>
+));
+
+EdgeModelRegistryInner.displayName = 'EdgeModelRegistryInner';
+
+const EdgeModelRegistry: React.FC = () => {
+  const { loaded } = useWatchComponents(true);
```

Comment:
why are you using useWatchComponents here?

This should be fetching the created models